### DISCUSSION TITLE
Internet Banking Blocked

### DISCUSSION THREAD

**Discussion by danielfr1982** [Ylianst/MeshCentral#6156] [Issues]
Hello, I believe that after updating the security guardian that banks use to secure the connection, it identifies the mesh agent installed and running and does not allow the user to continue accessing their account through the internet banking app. Has anyone seen this type of problem and if so, how to solve it? Thanks.

translation of what in the attached image:

authentication
It is not safe to proceed with the operation, I terminate remote access on your device.
Itaú does not request remote access to resolve problems with your account.

Find out more about security at itau.com.br/seguranca

![d8075ee41](https://github.com/Ylianst/MeshCentral/assets/151661210/a3ab53ee-9c5f-4524-a694-f701787ad6ca)

**Comment by si458** [comment]
I've seen this type of thing for exam software, we get asked a lot HOW TO BYPASS but in reality, you shouldn't be using remote support software/have it installed anyways for exams.

As for banking, that's a new one! Never seen it before?

Only thing u could try is customise the agent with your config.json, reinstall the meshagent (it will be called whatever u customised it to), see if the banking app picks it up?

Do u have it installed on that comp so u can remote control it for the end users support?

**Reply by danielfr1982** [reply]
Hi si458, yes, the agent was already installed on the computer belonging to a client I support and this is the first time the bank alert appeared. I believe it happened after updating the security guard that the bank uses.

**Reply by danielfr1982** [reply]
By the way, after uninstalling the agent, the user was able to access internet banking.

**Reply by si458** [reply]
Yeh, sadly, because people have used and abused meshcentral, most security companies/antiviruses detect the meshagent as malicious, even tho it can be used for genuine purposes!

We had to buy a security certificate which cost like £1k for 3 years, so we could customise the exe and sign it so antivirus software was happy that it wasn't malicious!

**Reply by danielfr1982** [reply]
Ok, I'll try customizing the agent and get back to you with more information.

**Reply by si458** [reply]
Change things like it's name, service name, company name, file description, icon? https://ylianst.github.io/MeshCentral/meshcentral/customization/#agent-branding

**Reply by danielfr1982** [reply]
Hello si458, I made the suggested changes and in the first test with the agent running the user was able to access internet banking. I will do more tests and come back with more information.

**Reply by si458** [reply]
well thats good news! as explained i think because of bad people, the name 'MeshCentral' and 'Mesh Agent' has just been blacklisted/marked as viruses/hackattacks/scams, which just isnt the case!
but thats why thankfully @Ylianst added in the support to allow customisation to at least try and help but also look professional at the same time haha

**Reply by danielfr1982** [reply]
Perfect. Thanks to you and Ylian for your support. Long live meshcentral. \m/
